### PR TITLE
STYLE: change the color of the output of a script

### LINKED ISSUE #44: Better way of separating the stdout in the notebook style examples
When #36 is merged, it would be good to find a better way to separate stdout from the input.

sphinx_gallery_output

Not sure about the best way how to do that at the moment, but just putting "Script output:" doesn't seem to cut it for me ...

### PR DESCRIPTION
**Pull request by GaelVaroquaux**
Fixes #44

### CODE REVIEW

File: sphinx_gallery/_static/gallery.css [hunk 1]
```diff
@@ -55,7 +55,9 @@ div.sphx-glr-thumbContainer {
 
 .sphx-glr-script-out pre {
   overflow-y: auto;
-  max-height: 30em; }
+  max-height: 30em;
+  background-color: #FFFFFE;
```
Comment (lesteve) [inline sphinx_gallery/_static/gallery.css]: This is almost white and I thought you wanted it light yellow ... did you mean #FFFFEF instead?

Comment (GaelVaroquaux) [inline sphinx_gallery/_static/gallery.css]: It looks nice on my screen with my eyes. I don't mind it to be very light.
screenshot

**Comment by Titan-C** [comment]
Back when I was thinking on issue #44 it came to my mind to have the script output with some indentation.

**Comment by GaelVaroquaux** [comment]
OK, new iteration taking in account this comment. I think that this iteration is indeed better. I reduced also the vertical spacing.
screenshot

**Comment by GaelVaroquaux** [comment]
The previous version disappearred, as it was in a diff that got overridden. I am putting it here for comparison.

screenshot_old

**Comment by lesteve** [comment]
Not a CSS expert but this looks fine to me.

**Comment by GaelVaroquaux** [comment]
One more iteration, to look better with the default sphinx CSS:
screenshot_default_css

**Comment by GaelVaroquaux** [comment]
One last thing: I have made the download more visible (a bit of CSS, and adding a div to the HTML):
screenshot_download

**Comment by GaelVaroquaux** [comment]
@Titan-C : how do you feel about this? Do you think that we can merge this, or do you have any further comments?

**Comment by Titan-C** [comment]
I'll try to nitpick. It is probably fine in the old default of sphinx. But the new default where everything is in white: code output and the code are not so different. The same for the read the docs theme. The slight indent helps a bit.

I like making the download, something closer to a button. I think it would be very useful in that case to actually copy the CSS trick I implemented for the gallery thumbnails to make the link overtake the full surface of its container making everything clickable, and friendlier for screens in mobile devices.

Finally as I confessed in #73 I actually code in SCSS, but I can port back your CSS style changes at a later time.

**Comment by GaelVaroquaux** [comment]
> I'll try to nitpick. It is probably fine in the old default of sphinx.
> But the new default where everything is in white: code output and the
> code are not so different. The same for the read the docs theme. The
> slight indent helps a bit.

OK, I'll have a lot. It's a pity that compiling the docs locally does not
show this.

> I like making the download, something closer to a button. I think it would be
> very useful in that case to actually copy the CSS trick I implemented for the
> gallery thumbnails to make the link overtake the full surface of its container
> making everything clickable, and friendlier for screens in mobile devices.

I'll look.

> Finally as I confessed in #73 I actually code in SCSS, but I can port
> back your CSS style changes at a later time.

I think that this is a problem. The files under version control should
always be editable. The way I work to do CSS is that I play with an
online editor (the chrome one, but it doesn't matter). However, I port
back manually the changes to the CSS. It helps having a clear and
readable CSS, which is important for long term maintenance.

**Comment by GaelVaroquaux** [comment]
New version of the color, in a brighter yellow, to make a difference with the read the docs theme. The screenshot attached is with a choice of colors that mimics this theme.

new_screen_yellow

**Comment by GaelVaroquaux** [comment]
> > I like making the download, something closer to a button. I think it
> > would be very useful in that case to actually copy the CSS trick I
> > implemented for the gallery thumbnails to make the link overtake the
> > full surface of its container making everything clickable, and
> > friendlier for screens in mobile devices.

> I'll look.

To do think, I would need to have only the download link, and not the
'Download Python source code' label. We could choose to keep only this
label, and loose the name of the file. The technical constrain is that I
cannot have 2 different styles in the button. 

I think that I prefer the current layout, that makes it clear what the
file name is. Indeed, on some system, the file just gets downloaded in
the background, and people might be looking for this file in the list of
files that they have downloaded.

**Comment by GaelVaroquaux** [comment]
To summarize, I think that I have addressed your comments as well as I could for now, and I'd like a new review for merge.

**Comment by agramfort** [comment]
LGTM

+1 for merge.

thanks @GaelVaroquaux

**Pull request merged**